src/gui/presets.cpp

```diff
@@ -931,6 +986,95 @@ void FurnaceGUI::initSystemPresets() {
       0
     }
   ));
+  cat.systems.push_back(FurnaceGUISysDef(
+    "Capcom CPS-1", { 
+      DIV_SYSTEM_YM2151, 64, 0, 2,
+      DIV_SYSTEM_MSM6295, 64, 0, 0,
+      0
+    }
+  ));
+  cat.systems.push_back(FurnaceGUISysDef(
+    "Jaleco Mega System 1", {
+      DIV_SYSTEM_YM2151, 64, 0, 2,
+      DIV_SYSTEM_MSM6295, 64, 0, 0,
+      DIV_SYSTEM_MSM6295, 64, 0, 0,
+      0
+    }
+  ));
+  cat.systems.push_back(FurnaceGUISysDef(
+    "NMK Arcade", {
+      DIV_SYSTEM_OPN, 64, 0, 0,
+      DIV_SYSTEM_MSM6295, 64, 0, 0,
+      DIV_SYSTEM_MSM6295, 64, 0, 0,
+      0
+    }
+  ));
+  cat.systems.push_back(FurnaceGUISysDef(
+    "Data East Arcade", {
+      DIV_SYSTEM_OPN, 64, 0, 0,
+      DIV_SYSTEM_OPL2, 64, 0, 0,
+      DIV_SYSTEM_MSM6295, 64, 0, 0,
+      0
+    }
+  ));
+  cat.systems.push_back(FurnaceGUISysDef(
+    "Kaneko Arcade", {
+      DIV_SYSTEM_AY8910, 64, 0, 1,
+      DIV_SYSTEM_AY8910, 64, 0, 1,
+      DIV_SYSTEM_MSM6295, 64, 0, 0,
+      0
+    }
+  ));
+  cat.systems.push_back(FurnaceGUISysDef(
+    "Tecmo Arcade", {
+      DIV_SYSTEM_OPN, 64, 0, 0,
+      DIV_SYSTEM_OPN, 64, 0, 0,
+      DIV_SYSTEM_MSM6295, 64, 0, 0,
+      0
+    }
+  ));
+  cat.systems.push_back(FurnaceGUISysDef(
+    "Seibu Kaihatsu Arcade", {
+      DIV_SYSTEM_OPL2, 64, 0, 0,
+      DIV_SYSTEM_MSM6295, 64, 0, 0,
+      0
+    }
+  ));
+  cat.systems.push_back(FurnaceGUISysDef(
+    "Data East Arcade (Dark Seal)", {
+      DIV_SYSTEM_YM2151, 64, 0, 2,
+      DIV_SYSTEM_OPN, 64, 0, 0,
+      DIV_SYSTEM_MSM6295, 64, 0, 0,
+      0
+    }
+  ));
+  cat.systems.push_back(FurnaceGUISysDef(
+    "Sunsoft Arcade", {
+      DIV_SYSTEM_YM2612, 64, 0, 4,
+      DIV_SYSTEM_MSM6295, 64, 0, 0,
+      0
+    }
+  ));
+  cat.systems.push_back(FurnaceGUISysDef(
+    "Atari Arcade (Rampart)", { //or Namco?
+      DIV_SYSTEM_OPLL, 64, 0, 0,
+      DIV_SYSTEM_MSM6295, 64, 0, 0,
+      0
+    }
+  ));
+  cat.systems.push_back(FurnaceGUISysDef(
+    "Data East Deco MLC System", {
+      DIV_SYSTEM_MSM6295, 64, 0, 0,
```

Comment:
Sound system for Data East MLC is single YMZ280B, runs at 14MHz.

[picture](https://www.emurom.net/images/emulation/autres_pcb/pcb/stadhr96.webp)

Reply:
Weird, Chain Reaction MLC uses M6295... https://vgmrips.net/packs/pack/chain-reaction-deco-mlc-system

Reply:
Ah, it's Deco 156 sys...

Reply:
> Weird, Chain Reaction MLC uses M6295... https://vgmrips.net/packs/pack/chain-reaction-deco-mlc-system

just misinformation in that link... Chain Reaction is not runs on MLC hardware, instead more cheaper one with 2 MSM6295s (1x2MHz w/ Bank and 1x1MHz w/o Bank).